pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "debian-dependency-bazelizer"
+version = "0"
+description = "A tool to turn the debian dependency graph into bazel modules"
+authors = ["Ziad Shaban <shaban_ziad@hotmail.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.8"
+boto3 = "^1.34.152"
+click = "^8.1.7"
+packaging = "^24.1"
+pytest = "^8.3.2"
+pytest-mock = "^3.14.0"
+# TODO: remove after update to ofiuco >=0.3.8. See
+# https://github.com/shabanzd/debian_dependency_bazelizer/pull/65#issuecomment-2264218951
+# https://github.com/shabanzd/debian_dependency_bazelizer/commit/e23fd203087c3df524623af48f4d8fe36a9c8a2e#r144963191
+urllib3 = "^1.26.0"
```

Comment:
I am not sure we need this either anymore

Reply:
nope, can remove it with boto3. poetry update is hanging for me, not sure what's up with that

Reply:
ah, it was https://github.com/python-poetry/poetry/issues/3365#issuecomment-1720309294 🤷